errors/errorHandler.js

```diff
@@ -0,0 +1,9 @@
+
+// eslint-disable-next-line no-unused-vars
+const errorHandler = (error, req, res, next) => {
+  const status = error.status || 500;
```

Comment:
Нужно исправить: обработчик ошибок отправляет неверный код.
Свойство `status` не определено в модуле ошибки:
https://github.com/Atm0Sphere94/project12.2/blob/3c0a8f4c5a52d4f0f5b4c115749023ff6621de1b/errors/NotFoundError.js#L1-L6

```suggestion
  const status = error.statusCode || 500;
```